app/views/submissions/show.html.erb

```diff
@@ -107,7 +107,7 @@
 
     <aside class="aside-widget aside-widget-absolute">
       <%= form_tag(submission_rate_path(submission), class: "submission-rating-form") do %>
-        <fieldset class="starability-fade fieldset-inline">
+        <fieldset class="starability-fade fieldset-inline" id="user-rating" data-rate="<%= current_user_rate_value %>">
```

Comment:
It's a nice custom to scope an attribute that's only a javascript handler with `js-` (here: `id="js-user-rating"`)